develop/index.md

```diff
@@ -33,3 +33,6 @@ most benefits from Docker.
 ## Advanced development with the SDK or API
 
 After you can write Dockerfiles or Compose files and use Docker CLI, take it to the next level by using Docker Engine SDK for Go/Python or use the HTTP API directly.
+
+- [Docker Engine SDK](../engine/api/sdk/){: target="_blank" rel="noopener" class="_"}
+- [HTTP API](../engine/api/latest/){: target="_blank" rel="noopener" class="_"}
```

Comment:
Thanks! Thinking we can;

- remove the "target=blank", because these links point to other locations in the documentation, and I don't think it's needed to open those in a new tab / window
- perhaps instead we should link to https://docs.docker.com/engine/api/, which (in the left-hand navigation) has links to both the SDK, the API reference, as well as providing a introduction to the API and links to SDK examples

Given that it would then be a single link, perhaps remove the bullet-list, and make it part of the paragraph itself;

```suggestion
After you can write Dockerfiles or Compose files and use Docker CLI, take it to
the next level by using Docker Engine SDK for Go/Python or use the HTTP API
directly. Visit the [Develop with Docker Engine API](../engine/api/index.md)
section to learn more about developing with the Engine API, where to find SDKs
for your programming language of choice, and to see some examples.
```

Reply:
Yes, makes more sense to link to the API section so users can see links to API and SDK docs.